crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/service/MaatCourtDataServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.justice.laa.crime.contribution.service;
```

Comment:
We could have some error/invalid params test cases here as these tests only covers the happy path.

Reply:
I'm not convinced this is necessary here as the underlying REST client is thoroughly tested in the crime-commons library. I can't think what else we would want to test? AFAIK this is how the service classes are tested in all of our modern services.